packages/hawtio/src/plugins/camel/icons/create-icon-index.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
if we go this direction, at least I'd like to have these scripts in the top level of the `@hawtio/react` package, not under `src/`, and make them runnable from yarn scripts.

and considering we'll need to maintain the scripts it might be better to have them in javascript/typescript instead of shell because we can then utilise the toolset such as linter and formatter for them as well.

Reply:
No removed as a result of refactoring the icons module